src/eradius_client.erl

```diff
@@ -664,6 +664,14 @@ find_suitable_peer(undefined) ->
     [];
 find_suitable_peer([]) ->
     [];
+find_suitable_peer([{Host, Port, Secret} | Pool]) when is_list(Host) ->
+    try
+	IP = get_ip(Host),
+	find_suitable_peer([{IP, Port, Secret} | Pool])
+    catch _:_ ->
+	% can't resolve ip by some reasons, just ignore it
```

Comment:
Maybe make sense to add log here for this case

Reply:
I don't see any sense in it